components/provisioner/internal/operations/stages/provisioning/create_operators_bindings.go

```diff
@@ -128,17 +129,25 @@ func (s *CreateBindingsForOperatorsStep) Run(cluster model.Cluster, _ model.Oper
 					administrator,
 					ownerClusterRoleBindingRoleRefName,
 					userKindSubject,
-					map[string]string{"app": "kyma", "type": "admin"}))
+					map[string]string{
+						"app":                                   "kyma",
+						"reconciler.kyma-project.io/managed-by": "reconciler",
+						"type":                                  "admin",
+					}))
 		}
 	}
 
 	if err := createClusterRoles(k8sClient.RbacV1().ClusterRoles(), clusterRoles); err != nil {
 		return operations.StageResult{}, err
 	}
 
+	//TODO: remove me after all clusters migrated
```

Comment:
I'd not add a TODO in the code :d It ends up living there for years. It's better to make a follow-up issue

Reply:
Fixed